styles.css

```diff
@@ -1,3 +1,13 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+@layer components {
+	.card {
+		@apply border border-gray-300 rounded-lg h-full m-2 lg:max-w-[325px] md:max-w-[360px] max-w-[500px] hover:shadow-lg;
+	}
+
+	.btn {
+		@apply font-bold px-4 py-2 mr-2 bg-red-400 text-white rounded-xl hover:bg-red-800 focus:bg-red-800 focus:border-2 focus:border-red-300;
+	}
```

Comment:
Your selector needs more specificity. This is how I see it in my browser:

<img width="1037" alt="Screen Shot 2023-03-31 at 16 09 12" src="https://user-images.githubusercontent.com/1735814/229208549-23b6cd72-e4dc-46be-96e9-41ecf3dffbae.png">